### ISSUE TITLE
Is there a way to minimize `script type="module"` part of an html?

### ISSUE THREAD

**Issue by frederikhors** [posthtml/htmlnano#134]
I have an .html file which contains this code:

```html
<script type="module">
  import { start } from "/./start.js";
  start({
    target: document.querySelector("body"),
    paths: { base: "", assets: "/." },
    session: {},
    host: location.host,
    route: true,
    hydrate: {
      status: 200,
      error: null,
      nodes: [import("/./app/pages\\index.js")],
      page: {
        host: location.host
        path: "/",
        query: new URLSearchParams(""),
        params: {},
      },
    },
    // ... and more
  });
</script>
```

posthtml with htmlnano does a good job for the html, but NOT for this `script type="module"` part.

Is there a way to minimize this?

**Comment by maltsev** [comment]
Thanks for @Scrum and his PR #135 now there is :-)

I'll release the new version of `htmlnano`. over the weekend.

**Comment by frederikhors** [comment]
> over the weekend

Why not immediately? :)

**Comment by maltsev** [comment]
> Why not immediately? :)

It takes some time to prepare a release.

**Comment by maltsev** [comment]
I released the new version 0.2.9.
